kafka/src/main/java/org/axonframework/extensions/kafka/configuration/KafkaMessageSourceConfigurer.java

```diff
@@ -41,11 +42,19 @@
     @Override
     public void initialize(Configuration config) {
         this.configuration = config;
-    }
 
-    @Override
-    public int phase() {
-        return Integer.MAX_VALUE;
+        if (!subscribableKafkaMessageSources.isEmpty()) {
```

Comment:
why is this not empty check needed?

Reply:
I didn't want to add `Runnables` that didn't do any real operations, as that seemed like a waste.
Earlier this wasn't an issue, as the `start` operation was invoked anyhow.
Right now, we're in control *if* anything is added to a given phase.
Hence I figured that checking if anything was present in the list saved some time.